BridgeCareApp/VuejsApp/src/components/scenarios/Scenarios.vue

```diff
@@ -152,5 +206,57 @@
         editSharedScenario() {
             this.$router.push({ path: 'EditScenario' });
         }
+
+        /**
+         * 'Run Simulation' button has been clicked
+         */
+        onRunSimulation(item: any) {
+            this.warning.showModal = true;
+            this.warning.heading = 'Warning';
+            this.warning.choice = true;
+            this.warning.message = 'The simulation can take around five minutes to finish. ' +
+                'Are you sure that you want to continue?';
+            this.currentItem = item;
+        }
+
+        /**
+         * A 'warning' modal decision has been made by the user
+         * @param value The user decision
+         */
+        onWarningModalDecision(value: boolean) {
+            this.warning.showModal = false;
+            if (value == true) {
+                this.runSimulation();
+            }
+        }
+
+        /**
+         * User has chosen to run a simulation
+         */
+        runSimulation() {
+            // dispatch action to run simulation
+            this.runSimulationAction({
```

Comment:
no setting of isBusy before dispatching to server

Reply:
Simulations run in the background. We are not freezing screen while a simulation is running. 
I have removed `isBusy = false` from `then` and `error` because I am never setting it to true